.github/CODEOWNERS

```diff
@@ -354,6 +355,8 @@ x-pack/examples/files_example @elastic/kibana-app-services
 /x-pack/test/functional/services/cases/ @elastic/response-ops
 /x-pack/test/functional_with_es_ssl/apps/cases/ @elastic/response-ops
 /x-pack/test/api_integration/apis/cases @elastic/response-ops
+/docs/api/cases @elastic/mlr-docs
+/x-pack/plugins/cases/docs/openapi @elastic/mlr-docs
```

Comment:
Do you mind if you put the `@elastic/response-ops` (`@elastic/response-ops @elastic/mlr-docs`) to the list to not block PRs that may change the `docs/api/cases` or the `openapi` folder? Do you think the PRs should be blocked until someone from the docs team approves?

Reply:
I think since we're transitioning to the open API specification as the source of truth and need to run commands to generate the output, it's necessary for writers to review. In the long term this process will get easier, but we're not there yet.

Reply:
Ok, @lcawl! Make sense to me.